core/feconf.py

```diff
@@ -22,11 +22,18 @@
 import datetime
 import enum
 import os
+import sys
 
 from core.constants import constants
 
 from typing import Dict, List, Union
-from typing_extensions import TypedDict
+
+# Add third_party to path. Some scripts access feconf even before
+# python_libs is added to path.
+_THIRD_PARTY_PATH = os.path.join(os.getcwd(), 'third_party', 'python_libs')
+sys.path.insert(0, _THIRD_PARTY_PATH)
+
+from typing_extensions import TypedDict # pylint: disable=wrong-import-position
```

Comment:
I wouldn't place this into _feconf.py_ as feconf is also imported in non-script files and it can mess up some things.

Reply:
Shifted to _utils.py_.